python/suricata/sc/suricatasc.py

```diff
@@ -118,9 +118,13 @@ def json_recv(self):
         data = ""
         while True:
             if sys.version < '3':
-                data += self.socket.recv(INC_SIZE)
+                received= self.socket.recv(INC_SIZE)
             else:
-                data += self.socket.recv(INC_SIZE).decode('iso-8859-1')
+                received= self.socket.recv(INC_SIZE).decode('iso-8859-1')
```

Comment:
nit: the style police will hit you with this missing space, and a few lines above.

Reply:
Oops ... I'll update